core/mondoo-azure-security.mql.yaml

```diff
@@ -815,42 +817,41 @@ queries:
       - uid: mondoo-azure-security-no-sql-databases-allow-ingress-0-0-0-0-0-api
     docs:
       desc: |
-        This check ensures that no SQL databases allow ingress connections from "0.0.0.0".
+        This configuration ensures that SQL databases across various platforms (Azure SQL, PostgreSQL, MySQL, MariaDB) do not permit unrestricted network access by blocking ingress from the IP address range "0.0.0.0/0". Limiting access is crucial to protect databases from unauthorized access and potential breaches. This setting is fundamental for enforcing the principle of least privilege, where only specific, approved IP addresses or ranges are permitted to access database resources, enhancing the security posture of Azure environments.
       audit: |
-        __cnspec run__
+        **Automated Auditing with Azure CLI**
 
-        To audit Microsoft Azure with `cnspec run`:
+        **Azure CLI:**
 
-        1. Run `az login` to authenticate with the correct Azure subscription and tenant.
-        2. Run this query:
+        List the firewall rules for all SQL servers and verify that none allow ingress from "0.0.0.0/0":
 
-          ```bash
-          cnspec run azure -c "azure.subscription.sql.servers.all(firewallRules.all(startIpAddress != "0.0.0.0"))"
-          cnspec run azure -c "azure.subscription.postgreSql.servers.all(firewallRules.all(startIpAddress != "0.0.0.0"))"
-          cnspec run azure -c "azure.subscription.mySql.servers.all(firewallRules.all(startIpAddress != "0.0.0.0"))"
-          cnspec run azure -c "azure.subscription.mariaDb.servers.all(firewallRules.all(startIpAddress != "0.0.0.0"))"
-          ```
+        ```bash
+        az sql server firewall-rule list --server <server-name> --resource-group <resource-group> --query "[?startIpAddress=='0.0.0.0'].{Name:name, StartIP:startIpAddress}"
+        az postgres server firewall-rule list --server <server-name> --resource-group <resource-group> --query "[?startIpAddress=='0.0.0.0'].{Name:name, StartIP:startIpAddress}"
```

Comment:
I removed "postgres" from the global dictionary since people tend to use this spelling, which is usually incorrect. Please add it locally for this repo.

Reply:
OK sure, Thanks